archinstall/lib/user_interaction.py

```diff
@@ -143,7 +143,17 @@ def ask_for_a_timezone():
 			level=LOG_LEVELS.Warning,
 			fg='red'
 		)
-		
+
+def ask_for_bootloader() -> str:
+	bootloader = "systemd-bootctl"
+	if hasUEFI()==False:
+		bootloader="grub-install"
+	else:
+		bootloader_choice = input("Would you like to use Grub as a bootloader over systemd-boot [y/N] ").lower()
```

Comment:
GRUB should be all caps, as it is an acronym. over should be "instead of"

Reply:
Just curious, is there a strong desire for GRUB over systemd-boot on a UEFI system? systemd-boot generally is the best solution for booting on UEFI.

Reply:
cool